### PR TITLE
FUSETOOLS2-1186 - changelog kind of config modeline as completion

### PR DESCRIPTION
**Pull request by apupier**

### CODE REVIEW

File: Changelog.md [hunk 1]
```diff
@@ -8,6 +8,7 @@
 - Update Camel Quarkus Catalog from 1.8.1 to 2.0.0
 - Adapt for Camel K community 1.5/Camel K Red Hat 1.4
   - Provide completion for new Camel K modeline option names
+  - Provide completion the different kind for `config` Camel K modeline option (`configmap`, `secret`, `file`)
```
Comment (lhein) [inline Changelog.md:11]: completion "for" ?

Comment (lhein) [reply Changelog.md]: I think it should be:
"Provide completion for the different kinds of `config` Camel K modeline option ..."

What do you think?

Comment (apupier) [reply Changelog.md]: I updated to almost that: https://github.com/camel-tooling/camel-lsp-client-vscode/pull/666/files#diff-c275fec9c453c2a42515bc5ab47e30fa4130ff1426aef2f6e000a9a34e122cb8R11

File: Changelog.md [hunk 2]
```diff
@@ -8,6 +8,7 @@
 - Update Camel Quarkus Catalog from 1.8.1 to 2.0.0
 - Adapt for Camel K community 1.5/Camel K Red Hat 1.4
   - Provide completion for new Camel K modeline option names
+  - Provide completion for the different kind of Camel K modeline option `config` (`configmap`, `secret`, `file`)
```
Comment (apupier) [inline Changelog.md:11]: proposals from @lhein:
"Provide completion for the different kinds of config Camel K modeline option ..."

Comment (apupier) [reply Changelog.md:11]: which is better. I will update

Comment (lhein) [reply Changelog.md]: yeah, but it needs to be "kinds" because there are actually 3 of them.

**Review by lhein — APPROVED** [review]

**Pull request merged**